lib/dl_api_commons/dl_api_commons/aiohttp/aiohttp_client.py

```diff
@@ -112,26 +106,24 @@ class BIAioHTTPClient:
 
     retrier: BaseRetrier = attr.ib(factory=NoRetriesRetrier)
 
-    ssl_context: Optional[ssl.SSLContext] = attr.ib(factory=default_ssl_context)
-    _session: Optional[aiohttp.ClientSession] = attr.ib()
-    close_session_on_exit: Optional[bool] = attr.ib(default=None)
+    _ca_data: bytes = attr.ib()
```

Comment:
why not to store str at the first place?

Reply:
Some libraries (for example grpc) wants ca_data as bytes (`grpc.ssl_channel_credentials`).

Thus using bytes is a clearer way for all cases (bytes from disk -> bytes in python -> bytes to grpc | decoded string to aiohttp | etc)